modules/extensions-react/src/hooks/useReplitEffect.ts

```diff
@@ -25,5 +25,5 @@ export default function useReplitEffect(
         };
       }
     }
-  }, [...dependencies, replit, status]);
+  }, [...(dependencies || []), replit, status]);
```

Comment:
usually when the dependency array is not specified, it means that every re-render causes a rerun of the callback. the empty dependency array would mean that no re-render would cause a rerun

Reply:
Ah, forgot about that.  Fixed.